api/src/main/java/org/eclipse/microprofile/concurrent/ThreadContext.java

```diff
@@ -240,6 +240,44 @@ public static Builder builder() {
      * scopes. An empty/default CDI context means that the thread does not have
      * access to the scope of the session, request, and so forth that created the
      * contextualized action.
+     * 
+     * For example, consider the following <code>&#64;RequestScoped<code> resource:
+     * 
+     * <code><pre>
+    &#64;RequestScoped
+    public class MyRequestScopedBean {
+        public String getState() {
+          // returns some request-specific information to the caller
+        }
+    }
+    </pre></code>
+     * 
+     * When CDI context is propagated, the request's state will also
```

Comment:
I would change it to something like:

_CDI context propagation includes request, session and conversation contexts.
When CDI context is propagated, all of the above mentioned contexts that are currently active will be available to the contextualized task with preserved state._